flatpak/flatpak-dotnet-generator.py

```
@@ -0,0 +1,121 @@
+                [
+                    "flatpak",
+                    "run",
+                    "--env=DOTNET_CLI_TELEMETRY_OPTOUT=true",
+                    "--env=DOTNET_SKIP_FIRST_TIME_EXPERIENCE=true",
+                    "--command=sh",
+                    f"--runtime=org.freedesktop.Sdk//{args.freedesktop}",
+                    "--share=network",
+                    "--filesystem=host",
+                    f"org.freedesktop.Sdk.Extension.dotnet{args.dotnet}//{args.freedesktop}",
+                    "-c",
+                    f'PATH="${{PATH}}:/usr/lib/sdk/dotnet{args.dotnet}/bin" LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/lib/sdk/dotnet{args.dotnet}/lib" exec dotnet restore "$@"',
```

Pylint (3.3.7)

[convention] Line too long (174/100)

(C0301)